<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPump</class>
 <widget class="QWidget" name="WidgetPump">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>452</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="12" column="0">
    <widget class="QLabel" name="label_port">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="spin_rate">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="10">
    <widget class="QListWidget" name="list_baud">
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QComboBox" name="combo_units">
     <item>
      <property name="text">
       <string>ml/min</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>µl/min</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ml/hr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>µl/hr</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" rowspan="10">
    <widget class="QListWidget" name="list_port"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="_label_2">
     <property name="text">
      <string>Baudrate</string>
     </property>
     <property name="buddy">
      <cstring>list_baud</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_baud">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="_label_1">
     <property name="text">
      <string>Port</string>
     </property>
     <property name="buddy">
      <cstring>list_port</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QPushButton" name="btn_send">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>97</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Send config</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="btn_get">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>97</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Get config</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QLabel" name="_label_3">
     <property name="minimumSize">
      <size>
       <width>97</width>
       <height>13</height>
      </size>
     </property>
     <property name="text">
      <string>Infuse rate</string>
     </property>
     <property name="buddy">
      <cstring>spin_rate</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QLabel" name="_label_5">
     <property name="minimumSize">
      <size>
       <width>97</width>
       <height>13</height>
      </size>
     </property>
     <property name="text">
      <string>Diameter</string>
     </property>
     <property name="buddy">
      <cstring>spin_diameter</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QDoubleSpinBox" name="spin_diameter">
     <property name="minimumSize">
      <size>
       <width>97</width>
       <height>20</height>
      </size>
     </property>
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>97</width>
       <height>13</height>
      </size>
     </property>
     <property name="text">
      <string>Target infusion</string>
     </property>
     <property name="buddy">
      <cstring>spin_target</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2">
    <widget class="QDoubleSpinBox" name="spin_target">
     <property name="minimumSize">
      <size>
       <width>97</width>
       <height>20</height>
      </size>
     </property>
     <property name="suffix">
      <string> ml</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="2">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="2" colspan="2">
    <widget class="QPushButton" name="btn_conn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>list_port</tabstop>
  <tabstop>list_baud</tabstop>
  <tabstop>btn_conn</tabstop>
  <tabstop>spin_rate</tabstop>
  <tabstop>spin_diameter</tabstop>
  <tabstop>spin_target</tabstop>
  <tabstop>btn_send</tabstop>
  <tabstop>btn_get</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
